### ISSUE TITLE
aria-current has incorrect value "true"

### ISSUE THREAD

**Issue by rmccue** [ReactTraining/react-router#6109]
## Version

4.2.2

## Test Case
https://codesandbox.io/s/wol5pjk8m5

(I couldn't actually get the original CodeSandbox link to work, but the code should be right.)

## Steps to reproduce

1. Add NavLink to page
2. Navigate to the page linked by that link
3. Note value of `aria-current` is `"true"`

## Expected Behavior

`aria-current` should have a [valid value](https://tink.uk/using-the-aria-current-attribute/); namely `"page"`

## Actual Behavior

`aria-current` has an invalid value; namely `"true"`

----

Hi! `aria-current` is an attribute which takes [one of a few valid string values](https://tink.uk/using-the-aria-current-attribute/). ~Currently in react-router-dom, [this is set](https://github.com/ReactTraining/react-router/blob/6d9c4250dd0b88e029e0698599b44aa8b0d8d90e/packages/react-router-dom/modules/NavLink.js#L45) to the value `"true"` instead.~ See comment; the error comes from the default being set to `"true"`.

This was caught during an accessibility audit of our site. :)

**Comment by rmccue** [comment]
Looking at this again, the `propTypes` are already set to the valid set of `aria-current` values; the issue is that [the default](https://github.com/ReactTraining/react-router/blob/6d9c4250dd0b88e029e0698599b44aa8b0d8d90e/packages/react-router-dom/modules/NavLink.js#L76) is incorrectly set to `"true"` instead of `"page"`.

**Comment by timdorr** [comment]
Yep, I brought this up the first time: https://github.com/ReactTraining/react-router/pull/4708/files#r120510832

A PR to change it would be appreciated!

**Comment by brandonrninefive** [comment]
Hello, I'd like to take this! I'll send over a PR shortly.

**Comment by brandonrninefive** [comment]
@timdorr @rmccue Correct me if I'm wrong, but based on the article linked above (https://tink.uk/using-the-aria-current-attribute/), it looks to me that `false` is also a valid option for `aria-current`. However, [this is not the case within NavLink](https://github.com/ReactTraining/react-router/blob/6d9c4250dd0b88e029e0698599b44aa8b0d8d90e/packages/react-router-dom/modules/NavLink.js#L64). In addition to setting `"page"` as the default, should I also add `"false"` as an option? Thanks!

**Comment by pshrmn** [comment]
@brandonrninefive I believe so. The [spec](https://www.w3.org/TR/wai-aria-1.1/#aria-current) includes `false`.

**Comment by rianrietveld** [comment]
Taking the liberty to cc @LJWatson in this conversation.
